src/main/python/interfaces/widgets/table.py

```diff
@@ -123,8 +136,13 @@ def if_select(self):
             return -1
 
     def get_cell(self, row: int, column: int):
+        """
+        :param row: required. row # of the cell
+        :param column: required. column # of the cell
+        :return: the text within the cell. 
+        """
         return self.item(row, column).text()
 
-    # TODO, when implement edit feature, cont work on this function
     def allow_edit(self, row: int):
```

Comment:
Just FYI: Edit functionality for resources has been implemented on the backend.

Reply:
I just look at it, and yes I think should be the time to implement it. And since this is the PR refactoring, let's leave that up in the next PR. It's on my checklist now.